packages/cli-platform-apple/src/commands/runCommand/runOptions.ts

```diff
@@ -0,0 +1,46 @@
+import {getDefaultUserTerminal} from '@react-native-community/cli-tools';
+
+export const runOptions = [
+  {
+    name: '--no-packager',
+    description: 'Do not launch packager while running the app',
+  },
+  {
+    name: '--port <number>',
+    default: process.env.RCT_METRO_PORT || 8081,
+    parse: Number,
+  },
+  {
+    name: '--terminal <string>',
+    description:
+      'Launches the Metro Bundler in a new window using the specified terminal path.',
+    default: getDefaultUserTerminal(),
+  },
+  {
+    name: '--binary-path <string>',
+    description:
+      'Path relative to project root where pre-built .app binary lives.',
+  },
+  {
+    name: '--list-devices',
+    description:
+      'List all available iOS devices and simulators and let you choose one to run the app. ',
```

Comment:
```suggestion
      'List all available platform-specific devices and simulators and let you choose one to run the app. ',
```
Or even better pass the platform name and use it there :)

Reply:
Let's leave this for the OOT platform to override. They can use the base runOptions and modify them. For example `macOS` won't have these options